UndertaleModTool/Scripts/Resource Repackers/ImportTileset.csx

```diff
@@ -0,0 +1,728 @@
+// Texture packer by Samuel Roy
+// Uses code from https://github.com/mfascia/TexturePacker
+// Entirely based off of code from ImportGraphics.csx script
+// Extended to import tilesets with proper tile IDs
+
+using System;
+using System.IO;
+using System.Drawing;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UndertaleModLib.Util;
+using System.ComponentModel;
+
+
+EnsureDataLoaded();
+
+bool importAsSprite = false;
+
+string importFolder = CheckValidity();
+
+string packDir = Path.Combine(ExePath, "Packager");
+Directory.CreateDirectory(packDir);
+
+string sourcePath = importFolder;
+string searchPattern = "*.png";
+string outName = Path.Combine(packDir, "atlas.txt");
+int textureSize = 2048;
```

Comment:
Make this 16384 or some other larger number - the texture packer is fine with working down in size, but will just skip over tilesets larger than 2048x2048 pixels, which is a lot of tilesets.